.infrastructure/cronjob.yml

```diff
@@ -0,0 +1,32 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: busybox-cronjob
+  namespace: mateapp
+spec:
+  schedule: "*/4 * * * *"
+  concurrencyPolicy: Allow
+  successfulJobsHistoryLimit: 10
+  failedJobsHistoryLimit: 5
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          namespace: mateapp
```

Comment:
The namespace 'mateapp' is specified again in the pod template metadata. Ensure consistency with the namespace specified in the metadata section of the CronJob.